packages/lesswrong/components/comments/CommentOnSelection.tsx

```diff
@@ -150,7 +151,9 @@ const SelectedTextToolbar = ({onClickComment, x, y, classes}: {
   classes: ClassesType,
 }) => {
   return <div className={classes.toolbar} style={{left: x, top: y}}>
-    <CommentIcon onClick={ev => onClickComment(ev)}/>
+    <AnalyticsContext pageElementContext="selectedTextToolbar">
+      <CommentIcon onClick={ev => onClickComment(ev)}/>
```

Comment:
I think you still have to put a `captureEvent` into the onClick handler. Other thing is would be good to track when people clicked submit as a result of this UI. That seemed less immediately obvious to me because the component is used elsewhere too.

Reply:
Final comment in submission happens inside `PopupCommentEditor`->`CommentsNewForm`. Connecting that attribution is somewhat complicated. Imo any comment-submission by someone who clicked the comment-on-selection button at all should count, even if it didn't come through the popup editor; starting a comment in the popup editor and then transferring to the below-the-post editor would happen if you refresh and continue from an autosave, or cut-and-paste into there.